src/Utils/ItemService.php

```diff
@@ -137,6 +145,42 @@ public function getItemFileList($itemId) {
         return [];
     }
 
+    /**
+     * Returns all searchable items contained in rooms specified by the given room IDs.
+     * @param integer[] $contextIds array of room IDs for rooms whose items shall be returned
+     * @return \cs_item[]
+     */
+    public function getSearchableItemsForContextIds(array $contextIds)
```

Comment:
Oder Alternativ \cs_list

```suggestion
    public function getSearchableItemsForContextIds(array $contextIds): array
```